perfkitbenchmarker/providers/openstack/os_disk.py

```diff
@@ -92,7 +92,8 @@ def Attach(self, vm):
         self.attached_vm_id = vm.id
 
         result = self.__nclient.volumes.create_server_volume(vm.id,
-                                                             self._disk.id,)
+                                                             self._disk.id,
+                                                             self.device)
```

Comment:
One question here.

If you noticed above in the `__init__` method for this class, `self.device = ""` is empty. Are you sure that's not a problem here?

According to the [docs](http://docs.openstack.org/developer/python-novaclient/ref/v2/volumes.html) `device` is optional, and at least, in KVM is not required at all, provided you have a somewhat recent kernel. [1]

[1]http://lists.openstack.org/pipermail/openstack-dev/2012-September/001117.html